smac/scenario/scenario.py

```diff
@@ -190,6 +190,9 @@ def _add_arguments(self):
                           dest='train_insts')
         self.add_argument(name='test_instances', default=[[None]], help=None,
                           dest='test_insts')
+        self.add_argument(name='initial_incumbent', default="DEFAULT",
+                          help=None, dest='initial_incumbent',
+                          callback=lambda arg: arg if arg in ['DEFAULT', 'RANDOM'] else None)
```

Comment:
This could pose a problem if a user provides the argument `DEFAULt`, it will be replaced by `None`, and the user will get an error message that the facade doesn't know what initial design None is. Also, I think we should make this case-insensitive.